packages/expo-video/ios/VideoManager.swift

```diff
@@ -40,19 +40,15 @@ class VideoManager {
     videoViews.remove(videoView)
   }
 
-  func onAppForegrounded() {
-    for videoPlayer in videoPlayers.allObjects {
-      videoPlayer.setTracksEnabled(true)
-    }
-  }
+  func onAppForegrounded() {}
 
   func onAppBackgrounded() {
     for videoView in videoViews.allObjects {
       guard let player = videoView.player else {
         continue
       }
       if player.staysActiveInBackground == true {
-        player.setTracksEnabled(videoView.isInPictureInPicture)
+        player.pointer.audiovisualBackgroundPlaybackPolicy = .continuesIfPossible
       } else if !videoView.isInPictureInPicture {
         player.pointer.pause()
```

Comment:
Let's set the audiovisualBackgroundPlaybackPolicy to `.pauses` when we don't need it. I assume keeping it in continuesIfPossible may have a negative effect on battery life.
```suggestion
        player.pointer.audiovisualBackgroundPlaybackPolicy = .continuesIfPossible
      } else if !videoView.isInPictureInPicture {
        player.ref.audiovisualBackgroundPlaybackPolicy = .pauses
        player.pointer.pause()
```

Reply:
Hi @behenate, could you please check if this will work?